src/components/suggestive_editor.rs

```diff
@@ -203,6 +202,21 @@ impl SuggestiveEditor {
         self.completion_dropdown.current_item()
     }
 
+    pub(crate) fn completion_all_filtered_dispatches(&self) -> Dispatches {
+        let mut final_dispatches = Dispatches::new(vec![]);
+
+        self.completion_dropdown
+            .filtered_item_groups()
+            .iter()
+            .for_each(|i| {
```

Comment:
Please avoid abbreviations for naming lambda parameter, for example:

Do:
```
items.map(|item| item.foo())
```

Don’t:
```
items.map(|i| i.foo())
```